Resizetizer.NT/Resizer.cs

```diff
@@ -38,33 +35,34 @@ public static string GetFileDestination(SharedImageInfo info, DpiPath dpi, strin
 			return destination;
 		}
 
-		public static ResizedImageInfo CopyFile(SharedImageInfo info, DpiPath dpi, string intermediateOutputPath, string inputsFile, ILogger logger, bool isAndroid = false)
```

Comment:
Not sure why this was static as half the parameters are actually in the constructor. Made an instance method to be consistent with

Reply:
Probably made sense initially and then I kept adding parameters and didn't consider making it instance :)